src/lib/Behat/Page/ContentUpdateItemPage.php

```diff
@@ -137,9 +138,26 @@ public function setExpectedContentDraftData(string $contentTypeIdentifier, strin
         $this->locationPath = $locationPath;
     }
 
-    public function getField(string $fieldName): FieldTypeComponent
+    public function getField(string $fieldName, ?int $fieldPosition = null): FieldTypeComponent
     {
-        $fieldLocator = new VisibleCSSLocator('', sprintf($this->getLocator('nthField')->getSelector(), $this->getFieldPosition($fieldName)));
+        if ($fieldPosition === null) {
+            $fieldPosition = $this->getFieldPosition($fieldName);
+        }
+
+        $activeSections = $this->getHTMLPage()
+            ->setTimeout(0)
```

Comment:
Shouldnt timeout be increased? I mean, it wont hurt to have like 5s timeout just in case.

Reply:
It's set to 0 on purpose - there is a chance that the view does not have any active sections and that's completely valid as well.

If there aren't any sections we don't want to spend 5 seconds searching for active sections, just continue with the test execution